features/importers/myhelsinki_places/tests/test_importer.py

```diff
@@ -1,6 +1,73 @@
+import datetime
```

Comment:
I would probably put the tests for importers in `features/tests/test_myhelsinki_places_importer.py`, to keep all tests in the same place. But it seems like a question of personal preference, so you can leave this as is too, if you prefer this way. 🙂